db/schema.rb

```diff
@@ -415,6 +415,13 @@
   add_index "users", ["authentication_token"], name: "index_users_on_authentication_token", unique: true, using: :btree
   add_index "users", ["login_id"], name: "index_users_on_login_id", unique: true, using: :btree
 
+  create_table "webcals", force: :cascade do |t|
```

Comment:
The migration defines the `webcals` table with a `string` primary key,

```ruby
create_table(:webcals, id: :string) do |t|
  t.boolean    :include_start_dates, null: false, default: false
  t.references :user, foreign_key: true, index: { unique: true }
end
```

...and `db:migrate` migrates it accordingly, but it doesn't get included in `schema.rb` and therefore `db:schema:load` implicitly defines `webcals#id` as an integer. 

https://github.com/rails/rails/issues/16599 is addresses this, but the fix isn't released in Rails 4.2